### PR TITLE
updated withdraw page ui

### PR DESCRIPTION
**Pull request by A6dulmalik**

### CODE REVIEW

**Comment by A6dulmalik** [comment]
@djeck1432 Kindly review this, thank you.

File: frontend/package-lock.json
Comment (whateverfw) [inline frontend/package-lock.json:1]: this should be removed

File: frontend/src/pages/vault/withdraw/withdraw.css
Comment (whateverfw) [inline frontend/src/pages/vault/withdraw/withdraw.css:1]: mobile page is completely broken? did you even test it before commit?

Comment (whateverfw) [inline frontend/src/pages/vault/withdraw/withdraw.css:1]: tablet version is also broken

Comment (whateverfw) [inline frontend/src/pages/vault/withdraw/withdraw.css:1]: mobile page background doesn't look like in figma at all

File: frontend/src/pages/vault/vaultLayout.css
Comment (whateverfw) [inline frontend/src/pages/vault/vaultLayout.css:1]: sidebar is now broken and to not always stick to the top when you scroll

File: frontend/yarn.lock
Comment (whateverfw) [inline frontend/yarn.lock:1]: I don't see any new packages, this file should've been changed, you should use `yarn install` and `yarn start` to run the project (if you do it locally)

File: frontend/src/pages/vault/withdraw/Withdraw.jsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ export default function Withdraw() {
     <VaultLayout>
     <div className="withdraw-wrapper">
       <div className="withdraw-container">
+        <h1 className="withdrawal-title">zkLend Withdraw</h1>
```
Comment (whateverfw) [inline frontend/src/pages/vault/withdraw/Withdraw.jsx:14]: this should not be visible in desktop, please take a careful look into figma

File: frontend/src/components/MetricCard/metricCard.css [hunk 2]
```diff
@@ -10,6 +12,18 @@
   gap: 4px;
 }
 
+.card {
+  width: 309px;
+  
+  background: rgba(18, 7, 33, 1);
+  border: 1px solid rgba(54, 41, 78, 1);
```
Comment (whateverfw) [inline frontend/src/components/MetricCard/metricCard.css:19]: colors should be used from css variables

**Review by whateverfw — CHANGES_REQUESTED** [review]

File: frontend/src/components/MetricCard/metricCard.css (hunk 2 above)
Comment (A6dulmalik) [reply frontend/src/components/MetricCard/metricCard.css:19]: I've made sure I did before pushing, would rectify.

File: frontend/src/pages/vault/withdraw/withdraw.css
Comment (whateverfw) [inline frontend/src/pages/vault/withdraw/withdraw.css:1]: right/left padding doesn't match figma

File: frontend/package-lock.json
Comment (whateverfw) [inline frontend/package-lock.json:1]: this file shouldn't exist, use yarn in a project

**Review by whateverfw — CHANGES_REQUESTED** [review]

File: yarn.lock
Comment (whateverfw) [inline yarn.lock:1]: remove this file from root folder

File: frontend/package-lock.json
Comment (whateverfw) [inline frontend/package-lock.json:1]: remove this file

File: frontend/src/pages/vault/withdraw/Withdraw.jsx [hunk 3]
```diff
@@ -4,20 +4,21 @@ import { ReactComponent as DiamondIcon } from 'assets/icons/diamond.svg';
...
-import { VaultLayout } from '../VaultLayout';
+// import { VaultLayout } from '../VaultLayout';
 
 export default function Withdraw() {
   return (
-    <VaultLayout>
+    // <VaultLayout>
     <div className="withdraw-wrapper">
       <div className="withdraw-container">
+       <h1 className="withdrawal-title">zkLend Withdraw</h1>
```
[the remark this answers is not in the document]
Comment (whateverfw) [reply frontend/src/pages/vault/withdraw/Withdraw.jsx:14]: still unresolved

File: frontend/src/pages/vault/withdraw/Withdraw.jsx [hunk 4]
```diff
@@ -4,20 +4,21 @@ import { ReactComponent as DiamondIcon } from 'assets/icons/diamond.svg';
...
         <div className="main-container">
           <div className="top-cards">
```
[the remark this answers is not in the document]
Comment (whateverfw) [reply frontend/src/pages/vault/withdraw/Withdraw.jsx:16]: still unresolved

File: frontend/src/pages/vault/withdraw/withdraw.css
Comment (whateverfw) [reply frontend/src/pages/vault/withdraw/withdraw.css:1]: still unresolved

File: frontend/src/pages/vault/vaultLayout.css
Comment (whateverfw) [inline frontend/src/pages/vault/vaultLayout.css:1]: sidebar is broken when scrolling, should be sticked to the top

**Review by whateverfw — CHANGES_REQUESTED** [review]
please make sure ui is matching figma, take a look into comments above

File: frontend/yarn.lock
Comment (whateverfw) [inline frontend/yarn.lock:1]: you shouldn't delete `yarn.lock` (that's fine that you deleted packaga-lock.json though)

**Review by whateverfw — CHANGES_REQUESTED** [review]
fix 2 remaining issues mentioned above, thanks

File: frontend/yarn.lock
Comment (A6dulmalik) [reply frontend/yarn.lock:1]: What are the two issues please, apart from the yarn.lock file?

Comment (whateverfw) [reply frontend/yarn.lock:1]: https://github.com/djeck1432/spotnet/pull/381#discussion_r1888050130

**Comment by A6dulmalik** [comment]
I've done that.
> @A6dulmalik Please, sync with main branch

**Review by whateverfw — APPROVED** [review]
@A6dulmalik @djeck1432 lgtm

**Review by djeck1432 — APPROVED** [review]
Looks good to me

**Comment by A6dulmalik** [comment]
That's great, looking forward to more contributions.
Thank you.

**Pull request merged**